### DISCUSSION TITLE
Questions around Text

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/GoOdf#97] [General]
Hi, would it be possible to put superscript text in the name of the stops? 

For example when I have 1st and 2nd trumpet, be able to put "st" and "nd" superscript..

Also, when adding images to a panel, is it possible to be able to add text to these images? I'm thinking of the labels indicating the name of each keyboard for example.. 

Finally, it is currently possible to apply a modification of font and font size to a set of stops... could we also apply a modification of color? 

Thank you again for all !

**Comment by larspalo** [comment]
Text characters must be supported by the character encoding (currently ISO-8859-1 is the only written, eventually I'll add or change to UTF-8 which allows for more characters). If you want any formatting or signs other than what exist as characters that can be written and interpreted in the .organ text file, the only way is to create it as images instead of text. But, no, as far as I've seen, no 2nd or 3rd are available as characters.

Images cannot have text, but labels can. You can simply create a label and assign it an image, to be used instead of the built-in versions, to able to have text written over it at.

It's perfectly possible to change text colour to any of the built-in word types or any html defined version by the colour picker. All such things can be copied to other similar GUI elemtents (with right-click).

**Comment by larspalo** [comment]
It seems like UTF-8 encoding should have superscript and subscript characters, but only a very limited number of them are supported in ISO-8859-1 so basically, it could quite possible be made to work (with UTF-8 that's supported within the app but not encoded correctly when written). But it's not really working right now.
